core/include/thread.h

```diff
@@ -434,6 +434,19 @@ char *thread_stack_init(thread_task_func_t task_func, void *arg, void *stack_sta
  */
 void thread_add_to_list(list_node_t *list, thread_t *thread);
 
+/**
+ * @brief Remove thread from list (internal)
```

Comment:
[`thread_add_to_list`](https://github.com/RIOT-OS/RIOT/pull/6158/files/94c15e16c70fec51df16af3f3dcd9256e22f2bb5#diff-9dcf1c99b916850378a11faa9f013f32R423) is not @internal. Should I update both?